packages/frontend/core/src/components/affine/setting-modal/setting-sidebar/index.tsx

```diff
@@ -145,14 +147,20 @@ export const SettingSidebar = ({
     [onTabChange]
   );
 
+  const isMobile = window.innerWidth <= 768;
```

Comment:
Can it be put in css, using media query + `display: none`

Reply:
Apologies for the delayed response. Thank you for the suggestion! I'll update the implementation accordingly.